server/api/person/person.subscribe.js

```diff
@@ -115,3 +138,7 @@ PubSub.subscribe(TOPIC_INTEREST__MESSAGE, async (msg, interest) => {
   await info
   PubSub.publish(TOPIC_PERSON__EMAIL_SENT, info)
 })
+
+function isSystemOperation (validationMessage) {
```

Comment:
suggest replacing with.
const isSystemOperation = validationMessage => validationMessage === SYSTEM_MEMBER_VALIDATION_MESSAGE

Reply:
I have removed this function.